[plugin-web-app] Add default values for web header and footer cut options (#4931)
vividus-framework/vividus: docs/modules/plugins/pages/plugin-applitools.adoc

|===

==== Properties based screenshot configuration for web applications

> Properties based screenshot configuration could be specified using `web.screenshot.strategy.<YOUR_STRATEGY_NAME>.<PROPERTY_NAME>= pattern`,

Reviewer: ```suggestion
Properties-based screenshot configuration can be specified using `web.screenshot.strategy.<YOUR_STRATEGY_NAME>.<PROPERTY_NAME>=<PROPERTY_VALUE>` pattern,
```

where:

> `YOUR_STRATEGY_NAME` - name of the custom strategy;
> `PROPERTY_NAME` - name of the property you want to specify.

Reviewer: ```suggestion
 - `YOUR_STRATEGY_NAME` - name of the custom strategy;
 - `PROPERTY_NAME` - name of the property you want to specify.
```

> Keep in mind that properties based configuration can be overridden by <<_visual_check_with_custom_applitools_and_screenshot_configuration, the step>> with custom step configuration.

Reviewer: ```suggestion
Keep in mind that properties-based configuration can be overridden by <<_visual_check_with_custom_applitools_and_screenshot_configuration, the step>> with custom step configuration.
```
